Sources/ValCommand/ValCommand.swift

```diff
@@ -84,6 +105,20 @@ public struct ValCommand: ParsableCommand {
       valueName: "output-type"))
   private var outputType: OutputType = .binary
 
+  @Option(
+    name: [.customLong("cc")],
+    help: ArgumentHelp(
+      "Customize the CXX compiler used by the Val backend. From: clang, gcc, msvc(Windows only)",
+      valueName: "CXXCompiler"))
+  private var customizeCXXCompiler: CXXCompiler = .clang
```

Comment:
```suggestion
  private var cxxCompiler: CXXCompiler = .clang
```
It's not necessarily custom; we'll always use this value IIUC.